database/insertData/_common/04_outcome_displays.js

```diff
@@ -81,11 +81,35 @@ exports.queries = [
                 title:  "Address"
               }
               {
-                columnName: "logo"
+                columnName: "logoUrl"
+                elementTypePluginCode: "imageDisplay"
+                isTextColumn: true
+                parameters: {
+                  url: {
+                    operator: "CONCAT"
```

Comment:
This works, but strictly speaking we were trying to deprecate "CONCAT" and just use "+". (You can just swap it out and it will do exactly the same thing)

Reply:
095dfa0bdd7973e6b49c981ba80c412462415d2a